services/API-service/src/api/event/event-place-code.entity.ts

```diff
@@ -0,0 +1,26 @@
+import { Entity, Column, Check, PrimaryGeneratedColumn } from 'typeorm';
+
+@Entity('event_place_code', { schema: 'IBF-pipeline-output' })
+export class EventPlaceCodeEntity {
+  @PrimaryGeneratedColumn()
+  public id: number;
```

Comment:
`id` here refers to `EventPlaceCodeId` ? then I would name it `eventPlaceCodeId`

Reply:
I dunno it's the id column of the EventPlaceCodeEntity so is id not sufficient?

Reply:
the properties get assigned to different variables when we use them elsewhere so it gets confusing when I try to interpret just "id"